src/fklearn/resources/VERSION

```diff
@@ -1 +1 @@
-1.16.1
+1.16.2
```

Comment:
Shouldn't it be 1.17.1 since there's a new feature instead of a bug fix?

Reply:
Sure, just did it. Didn't know how we're doing versioning :)